.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+cache:
+  yarn: true
+notifications:
+  email: false
+  webhooks: https://outlook.office.com/webhook/03fa4a79-572f-4c68-b756-e4e851d0215a@9093f1a3-8771-4fb7-8596-d51eeef18cda/TravisCI/449087910d3647598cf3d7c6387fb8fc/286c079f-6085-4aa0-8f8d-e2a3e8d1f568
+node_js:
+  - '10'
+# TODO: add semantic-release?
+# after_success:
+#   - semantic-release
```

Comment:
Probably no need for semantic-release either since we aren't really creating any formal releases.